src/components/App.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import ButtonPanel from './ButtonPanel';
+import Display from './Display';
+
+function App() {
+  return (
+    <div>
+      <Display result="0" />
```

Comment:
For this milestone  not required to set default value here for display component